docs/install-infra-on-k0s.md

```diff
@@ -0,0 +1,168 @@
+# Installing k0s with Infra
+
+Getting k0s to work with Infra requires a few extra steps to get working.
+Out of the box, k0s does not include a load balancer or persistent storage
+which are required to run Infra.
+
+We'll install:
+  * [Metallb](https://docs.k0sproject.io/head/examples/metallb-loadbalancer/)
+  * [OpenEBS](https://docs.k0sproject.io/head/storage/)
+
+This guide assumes that you've already installed the Infra CLI. You
+can check out the [Quickstart guide](https://infrahq.com/docs/getting-started/quickstart)
+which contains directions on how to install the CLI.
+
+### 1. Download k0s
```

Comment:
We should just link to the k0s guide here rather than re-iterating their install steps, that way we don't get out of sync:
https://docs.k0sproject.io/v1.21.2+k0s.1/install/

Reply:
The problem is there are some non-quickstart steps that need to be done first unfortunately. Since we'll post this in the discussion forum for now, I think it will probably be fine.